site/posts/writing-multiple-netcdf-files-in-parallel-with-xarray-and-dask.ipynb

```diff
@@ -0,0 +1,2084 @@
+       "\n",
+       "  <!-- Vertical lines -->\n",
+       "  <line x1=\"10\" y1=\"0\" x2=\"33\" y2=\"23\" style=\"stroke-width:2\" />\n",
+       "  <line x1=\"130\" y1=\"0\" x2=\"153\" y2=\"23\" style=\"stroke-width:2\" />\n",
+       "\n",
+       "  <!-- Colored Rectangle -->\n",
+       "  <polygon points=\"10.000000,0.000000 130.000000,0.000000 153.547663,23.547663 33.547663,23.547663\" style=\"fill:#ECB172A0;stroke-width:0\"/>\n",
+       "\n",
+       "  <!-- Horizontal lines -->\n",
+       "  <line x1=\"33\" y1=\"23\" x2=\"153\" y2=\"23\" style=\"stroke-width:2\" />\n",
+       "  <line x1=\"33\" y1=\"113\" x2=\"153\" y2=\"113\" style=\"stroke-width:2\" />\n",
+       "\n",
+       "  <!-- Vertical lines -->\n",
+       "  <line x1=\"33\" y1=\"23\" x2=\"33\" y2=\"113\" style=\"stroke-width:2\" />\n",
+       "  <line x1=\"153\" y1=\"23\" x2=\"153\" y2=\"113\" style=\"stroke-width:2\" />\n",
+       "\n",
+       "  <!-- Colored Rectangle -->\n",
+       "  <polygon points=\"33.547663,23.547663 153.547663,23.547663 153.547663,113.002208 33.547663,113.002208\" style=\"fill:#ECB172A0;stroke-width:0\"/>\n",
+       "\n",
+       "  <!-- Text -->\n",
+       "  <text x=\"93.547663\" y=\"133.002208\" font-size=\"1.0rem\" font-weight=\"100\" text-anchor=\"middle\" >275</text>\n",
+       "  <text x=\"173.547663\" y=\"68.274935\" font-size=\"1.0rem\" font-weight=\"100\" text-anchor=\"middle\" transform=\"rotate(-90,173.547663,68.274935)\">205</text>\n",
+       "  <text x=\"11.773831\" y=\"121.228377\" font-size=\"1.0rem\" font-weight=\"100\" text-anchor=\"middle\" transform=\"rotate(45,11.773831,121.228377)\">36</text>\n",
+       "</svg>\n",
+       "</td>\n",
+       "</tr>\n",
+       "</table></pre></li></ul></div></li><li class='xr-section-item'><input id='section-90961890-aac3-4dbf-899e-8e3e1c489a19' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-90961890-aac3-4dbf-899e-8e3e1c489a19' class='xr-section-summary'  title='Expand/collapse section'>Attributes: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.Dataset>\n",
+       "Dimensions:  (time: 36, x: 275, y: 205)\n",
+       "Coordinates:\n",
+       "    yc       (y, x) float64 dask.array<chunksize=(205, 275), meta=np.ndarray>\n",
+       "    xc       (y, x) float64 dask.array<chunksize=(205, 275), meta=np.ndarray>\n",
+       "  * time     (time) object 1980-09-16 12:00:00 ... 1983-08-17 00:00:00\n",
+       "Dimensions without coordinates: x, y\n",
+       "Data variables:\n",
+       "    Tair     (time, y, x) float64 dask.array<chunksize=(12, 205, 275), meta=np.ndarray>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "result = np.sqrt(np.sin(ds) ** 2 + np.cos(ds) ** 2)\n",
+    "result"
```

Comment:
Do you need to output `result` here?  It is a lot of text and it's HTML display is not much different than the first time it is displayed, above.  Perhaps you could just do:

```suggestion
    "result = np.sqrt(np.sin(ds) ** 2 + np.cos(ds) ** 2)\n"
```